devtools/cli/src/main/java/io/quarkus/cli/common/RegistryClientMixin.java

```diff
@@ -38,12 +49,9 @@ public ExtensionCatalog getExtensionCatalog(TargetQuarkusVersionGroup targetVers
                         QuarkusProjectHelper.artifactResolver(), log);
             }
 
-            if (targetVersion.isStream()) {
-                final String stream = targetVersion.getStream();
-                final int colon = stream.indexOf(':');
-                final String platformKey = colon <= 0 ? null : stream.substring(0, colon);
-                final String streamId = colon < 0 ? stream : stream.substring(colon + 1);
-                return catalogResolver.resolveExtensionCatalog(platformKey, streamId);
+            if (targetVersion.isStreamSpecified()) {
+                final StreamCoords stream = targetVersion.getStream();
+                return catalogResolver.resolveExtensionCatalog(stream.getPlatformKey());
```

Comment:
Is there a reason the streamId isn't passed here?

Reply:
nope. oversight.